File: HelloWorld.java
package IntroProg;

import java.util.Scanner;

class HelloWorld {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter your password");

        String password = in.nextLine();
        String confirm = in.nextLine();
        if (password == confirm)
            System.out.println("Welcome aboard");
        
    }
}

File: HelloWorld.py
def login(name,password):
    print ("logged in")
def register(name,password):
    print ("registered")

def begin():
    print("Welcome to Password Generator login")
    option = input("login or registor (login,reg): ")
    if(option!="login" and option!="reg"):
        option = begin()
    return option
opt = begin()
def GetInfo():
    if(opt=="login"):
        name = input("Enter your name: ")
        password = input("Enter your password: ")
        login(name,password)
    else: 
        print("Enter your name and password to register")
        name = input("Enter your name: ")
        password = input("Enter your password: ")
        register(name,password)
